cmake/FindNetCDF.cmake

```diff
@@ -1,8 +1,3 @@
-# (C) Copyright 2017-2020 UCAR
-#
-# This software is licensed under the terms of the Apache Licence Version 2.0
-# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
-#
 # (C) Copyright 2011- ECMWF.
```

Comment:
Have you compared this file to https://github.com/NOAA-EMC/CMakeModules/blob/develop/Modules/FindNetCDF.cmake ? This is what we use and maintain in jedi-cmake and spack-stack (as a git submodule).

Reply:
They're the same.